modules/xmpp/SDP.js

```diff
@@ -71,23 +71,20 @@ SDP.prototype.getMediaSsrcMap = function() {
     }
     return media_ssrcs;
 };
+
 /**
- * Returns <tt>true</tt> if this SDP contains given SSRC.
- * @param ssrc the ssrc to check.
- * @returns {boolean} <tt>true</tt> if this SDP contains given SSRC.
+ * Returns <tt>true</tt> if this SDP contains a given SSRC.
+ * @param ssrc the SSRC to look for in this SDP.
+ * @returns {boolean} <tt>true</tt> if this SDP contains the given SSRC.
  */
 SDP.prototype.containsSSRC = function (ssrc) {
-    // FIXME this code is really strange - improve it if you can
-    var medias = this.getMediaSsrcMap();
-    var result = false;
-    Object.keys(medias).forEach(function (mediaindex) {
-        if (result)
-            return;
-        if (medias[mediaindex].ssrcs[ssrc]) {
-            result = true;
-        }
-    });
-    return result;
+    var media = this.getMediaSsrcMap();
+    for (var i = 0, length = media.length; i < length; ++i) {
```

Comment:
"media" is JS object and it seems that length property is undefined. Maybe we should use and return array in getMediaSsrcMap method. That way length won't be undefined.